### PR TITLE
Added Setup script

### PR DESCRIPTION
**Pull request by photon1503**
Since the end user needs to download 2 components from different GitHub repositories, and we have a dependencies, some might struggle.

Therefore I created a setup script for [Inno Setup](https://jrsoftware.org/isinfo.php) which will generate one executable setup file.

Currently the script needs to be executed locally, but there are also GitHub actions available which we could integrate.

Basically the setup is doing the following steps:

- Copy the DLL to the users's simhub folder
- Download the Dashboard from GitHub, with a given version
- Execute the downloaded Dashboard to force the SimHub installation.

image
image
image

### CODE REVIEW

File: Setup/innosetup.iss [hunk 1]
```diff
@@ -0,0 +1,58 @@
+; -- Example1.iss --
+
+[Setup]
+AppName=Dahl Design Dahsboard & Plugin
```
Comment (andreasdahl1987) [inline Setup/innosetup.iss:4]: Dashboard

**Comment by andreasdahl1987** [comment]
This is a very good addition, you might have noticed a few people are struggling to get it up and running

**Comment by andreasdahl1987** [comment]
Tested and working for me :) 

How will the workflow be here? Will it always fetch the latest release? Or will it have to be adjusted on new releases? Could we add the LED plugin .dll in there as well? 

I noticed there will be a dashboard file sitting in the main simhub folder after installation. Is there a way to remove it in the installation process?

**Comment by photon1503** [comment]
Regarding workflow, it will fetch a dedicated version (not _lastest_) otherwise the DLLs and the dashboard won't fit together.

I added some constants at the top of the InnoSetup script which will be used to fetch the correct files.

Also the .simhubdash file is now deleted after installation

**Comment by photon1503** [comment]
Now I got the point of your question. We are embedding the Plugin DLL into the installation EXE, but download the other files which is inconsistant.

Changed this to download all components from web, so we now have a "Web Installer" :-)

**Comment by andreasdahl1987** [comment]
Really good! What I'm thinking about is wether to have this installer here on the properties repo or on the dash repo. Some use the properties plugin alone, just for the properties. Some just want the dashboard and don't know about dependencies. The properties plugin is the only one in my collection that is completely dependency free. I'm also working on a gitbook for the dashboard which will be connected to the dash repo.

What do you think? Have install.exe in the dash repo and downloadable .dlls in prop/LED repos?

**Comment by photon1503** [comment]
yes maybe better to move it to the DashBoard Repo, since we have the dependency there to the Plugin but not the other way around.

I'll close this PR and move the files accordingly

**Pull request closed without merging**